fabricadesoftware-ifc/fabricaNode-BackEnd
# FIX: ManyToMany Publicacao_Keyword

### Fix field n:n keyword on publicacao

**Description:**
This issue fixing the many-to-many keyword field of the publicacao model

**Test:**

1. create a superuser
2. Initialize the server
3. access the endpoint: "/admin"
4. create a publication with the keywords
5. check if everthing is ok

**Have you fulfilled the entire issue description?**
